index.bs

```diff
@@ -1269,7 +1269,9 @@ The dictionary SHALL have the following fields:
 # Interface `ReceiveStream` #  {#receive-stream}
 
 A <dfn interface>ReceiveStream</dfn> is a {{ReadableStream}} of {{Uint8Array}}
-that can be read from, to consume data received from the server.
+that can be read from, to consume data received from the server. {{ReceiveStream}} is a
```

Comment:
@domenic What do you think about this explanation? If this looks good, would you export the terms in https://github.com/whatwg/streams/pull/1130? Or is it better for me to create a separate PR?

Reply:
This explanation looks great. I did not realize those terms were not exported; I will do so in the PR.